### ISSUE TITLE
Disable remote-controlled window title changing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#385]
```
There is an option in PuTTY to disable remote-controlled window title changing. 
This is particularly helpful to me in some cases where I specify the title in 
the command line and want the window title to be locked as such. Mintty lets me 
specify the title, but happily overwrites it when it receives the proper escape 
sequence from a server.

A configuration option would be nice, but even something as simple as 
automatically using this behavior when the -t option was specified in the 
command line would be a nice improvement.
```

Original issue reported on code.google.com by `ryan.kac...@gmail.com` on 24 May 2013 at 11:32

**Comment by mintty** [comment]
Added option -T or --Title=...

**Comment by mintty** [comment]
Released 2.9.1.
